test/test-helper.el

```diff
@@ -115,19 +115,23 @@ failures.
 Inspired by textwrap.dedent, in Python."
   (let ((margin (apply #'min
                        (mapcar (lambda (l)
-                                 (or (string-match "[^ ]+" l) 0))
+                                 (or (string-match "[^ ]+" l) 10000000))
```

Comment:
Why does this work, again?

Reply:
This is giving the index of the first non-space character. If there is no non-space character, we can effectively ignore this line